pkg/volume/volume.go

```diff
@@ -240,6 +246,9 @@ type DeviceMounter interface {
 	// individual pods can then bind mount
 	// Note that devicePath can be empty if the volume plugin does not implement any of Attach and WaitForAttach methods.
 	MountDevice(spec *Spec, devicePath string, deviceMountPath string) error
+
+	// MountDeviceWithStatusTracking is same as MountDevice except status of mount operation is also returned
+	MountDeviceWithStatusTracking(spec *Spec, devicePath string, deviceMountPath string) (volumetypes.OperationStatus, error)
```

Comment:
Okay, so I ended up removing both of `*WithStatusTracking` functions and using plain `MountDevice` and `SetUp`.  I did not like the idea of introducing new interface because logically it did not fit in with existing interfaces.  

This does mean - surface area of the change will be bigger.